lib/dndcolors.sty

```diff
@@ -1,52 +1,67 @@
 
 % Type
-\definecolor{titlered}{HTML}{58180D}	% titles
-\definecolor{titlegold}{HTML}{C9AD6A} % titlerules
-\definecolor{rulered}{HTML}{9C2B1B}		% triangular rule in statsblock
+\definecolor {titlered}  {HTML} {58180D} % titles
+\definecolor {titlegold} {HTML} {C9AD6A} % titlerules
+\definecolor {rulered}   {HTML} {9C2B1B} % triangular rule in statsblock
 
 % Trim (affects tables and paperboxes)
-\definecolor{PhbLightGreen}{HTML}{E0E5C1} % PHB Part 1
-\definecolor{PhbLightCyan}{HTML}{B5CEB8}  % PHB Part 2
-\definecolor{PhbMauve}{HTML}{DCCCC5}      % PHB Part 3
-\definecolor{PhbTan}{HTML}{E5D5AC}        % PHB appendix
-\definecolor{DmgLavender}{HTML}{E3CED3}   % DMG Part 1
-\definecolor{DmgCoral}{HTML}{F3D7C1}      % DMG Part 2
-\definecolor{DmgSlateGray}{HTML}{DBE4E4}  % DMG Part 3
-\definecolor{DmgLilac}{HTML}{D7D4D6}      % DMG appendix
+\definecolor {PhbLightGreen} {HTML} {E0E5C1} % PHB Part 1
+\definecolor {PhbLightCyan}  {HTML} {B5CEB8} % PHB Part 2
+\definecolor {PhbMauve}      {HTML} {DCCCC5} % PHB Part 3
+\definecolor {PhbTan}        {HTML} {E5D5AC} % PHB appendix
+\definecolor {DmgLavender}   {HTML} {E3CED3} % DMG Part 1
+\definecolor {DmgCoral}      {HTML} {F3D7C1} % DMG Part 2
+\definecolor {DmgSlateGray}  {HTML} {DBE4E4} % DMG Part 3
+\definecolor {DmgLilac}      {HTML} {D7D4D6} % DMG appendix
 
 % Trim aliases
-\colorlet{DmgSlateGrey}{DmgSlateGray}
+\colorlet {DmgSlateGrey} {DmgSlateGray}
 
 % The color used in \setthemecolor when a new color is not set
-\colorlet{themecolor}{PhbLightGreen}    % Set the default theme to Part 1 of the PHB.
+\colorlet {themecolor} {PhbLightGreen} % Set the default theme to Part 1 of the PHB.
 
 % Element colors that change when \setthemecolor is used
-\colorlet{commentboxcolor}{themecolor}  % commentbox background
-\colorlet{paperboxcolor}{themecolor}    % paperbox background
-\colorlet{tablecolor}{themecolor}       % table even row background
+\colorlet {commentboxcolor} {themecolor} % commentbox background
+\colorlet {paperboxcolor}   {themecolor} % paperbox background
+\colorlet {tablecolor}      {themecolor} % table even row background
 
 % Element colors that do not respond to \setthemecolor
-\colorlet{quoteboxcolor}{bgtan}             % quotebox background
-\definecolor{statblockribbon}{HTML}{E69A28} % stat block top/bottom borders (gold)
-\definecolor{statblockbg}{HTML}{FDF1DC}     % stat block background (tan)
+\colorlet {quoteboxcolor} {bgtan} % quotebox background
+\definecolor {statblockribbon} {HTML} {E69A28} % stat block top/bottom borders (gold)
+\definecolor {statblockbg}     {HTML} {FDF1DC} % stat block background (tan)
 
 % Sets the themecolor and colors for all themed elements
-% If called without the optional color, resets the color of all themed elements to the current themecolor
+% If called without the optional color, resets the color of all themed elements to the current themecolor  
+% Old interface
 \newcommand{\setthemecolor}[1][themecolor]{%
```

Comment:
Let's stick a deprecation notice in here (for 1.0).

The new deprecation macro technically doesn't exist yet, so we can use the old one for now. We can replace all the remaining `dnd@deprecate` calls when we clean up `new-interface`.

Reply:
I moved all of the deprecate commands to a separate branch to merge in after all the different parts are integrated.

Reply:
Sure, that works.